programs/stake/src/stake_state.rs

```diff
@@ -384,8 +394,9 @@ impl Stake {
         point_value: &PointValue,
         vote_state: &VoteState,
         stake_history: Option<&StakeHistory>,
+        tracer: &mut Option<impl FnMut(&InflationPointCalcEvent)>,
```

Comment:
I'd prefer a more descriptive name for `tracer`: `inflation_point_calc_tracer`?

Reply:
done: 41c8d0bb11